ankh/main.go

```diff
@@ -305,9 +306,8 @@ func promptForMissingConfigs(ctx *ankh.ExecutionContext, ankhFile *ankh.AnkhFile
 		}
 
 		// we should finally have a tag value
-		if ctx.SlackChannel != "" {
-			ctx.SlackDeploymentVersion = *chart.Tag
-		}
+		ctx.DeploymentVersion = *chart.Tag
```

Comment:
we should rename `DeploymentVersion` to `DeploymentTag` to disambiguate with chart `Version`